fast/addons/2-networking-swp/main.tf

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  aliased_project_id = lookup(
+    var.host_project_ids, var.project_id, var.project_id
+  )
+  project_id = try(
+    module.project[0].project_id, var.project_id
+  )
+  regions = {
+    for k, v in var.locations : k => lookup(var.regions, v, v)
+  }
+  swp_configs = flatten([
+    for r_k, r_v in local.regions : [
+      for k, v in var.swp_configs : merge(v, {
+        name             = "${var.name}-${k}-${r_k}"
+        region           = r_v
+        region_shortname = r_k
+      })
+    ]
+  ])
+}
+
+module "project" {
+  source         = "../../../modules/project"
+  count          = var._fast_debug.skip_datasources == true ? 0 : 1
+  name           = local.aliased_project_id
+  project_create = false
+  service_agents_config = {
+    services_enabled = [
```

Comment:
Should this be guarded in the same way you have `services` below? Something like
```hcl
  services_enabled = var.enable_services != true ? [] : [
    "networksecurity.googleapis.com"
  ] 
```